roles/tower/templates/tower_web.yaml.j2

```diff
@@ -94,10 +94,12 @@ spec:
   - port: 80
     protocol: TCP
     targetPort: 8052
+    name: http
   selector:
     app: tower
 
 # Tower Ingress.
+{% if 'ingress' == tower_ingress_type %}
```

Comment:
We need to document `tower_ingress_type` in README (and I'd prefer setting them in https://github.com/geerlingguy/tower-operator/blob/master/deploy/crds/tower_v1alpha1_tower_cr_awx.yaml and https://github.com/geerlingguy/tower-operator/blob/master/deploy/crds/tower_v1alpha1_tower_cr_tower.yaml to be complete).

Reply:
:+1: